### PR TITLE
qa-vdb: add limitation notes

### PR DESCRIPTION
**Pull request by kuraga**
See also: #15.

### CODE REVIEW

File: scripts/qa-vdb [hunk 1]
```diff
@@ -67,6 +67,10 @@ usage <<-EOU
 	Missing slots won't be displayed when a binding operator (:=) is used as
 	portage records the slot even if it was missing from the ebuild.
 
+    Installed packages' files are considered as candidates only.
+
+    There won't be a warning if a library isn't provided by any package.
```
Comment (ionenwks) [inline scripts/qa-vdb:72]: Indenting here needs to be tabs, or it won't be aligned with the above block in output. Given this is about 1 same limitation I wouldn't put a blank line in between too.

imo wouldn't hurt to remind that portage can warn too, maybe do this?
```
	No warning is given in the event libraries are missing entirely or not
	associated with a package. For the former, please use portage's default
	enabled FEATURES="qa-unresolved-soname-deps" to be informed.
```

File: README.rst [hunk 2]
```diff
@@ -62,6 +62,9 @@ Note that "unused" dependencies should be taken with a grain of salt, package
 may or may not still need it in some other way than ``DT_NEEDED``. ``dlopen()``,
 build time checks/headers, executables, and other potential non-library files.
 
+Note that installed packages' files are considered as candidates only.
+Note that there won't be a warning if a library isn't provided by any package.
```
Comment (ionenwks) [inline README.rst:66]: For the sake of not having too much redundant information, I did commit 80ddaa8aac3290051de86d5c8fb6fbd504ebdea7 -- so I'd skip this.

**Review by ionenwks — CHANGES_REQUESTED** [review]
Good idea, wouldn't hurt to note it.

File: scripts/qa-vdb (hunk 1 above)
Comment (kuraga) [reply scripts/qa-vdb:72]: `in` == `if`?

File: README.rst (hunk 2 above)
Comment (kuraga) [reply README.rst:66]: Then we could remove the previous paragraph, either :smile: (a joke)

**Comment by kuraga** [comment]
@ionenwks , updated! Thanks!

File: scripts/qa-vdb (hunk 1 above)
Comment (ionenwks) [reply scripts/qa-vdb]: No? Could use `if` if did "No warning is given if libraries are missing entirely ..", but "if the event" doesn't work without `in`. Not that I'm a reference for using English properly.

Also, the previous line about installed files is still there (seems redundant?).

Comment (kuraga) [reply scripts/qa-vdb]: Ok, my English is not perfect, sorry :)

**Comment by kuraga** [comment]
@ionenwks ,updated. Add `installed`, review, please.

But... No warning is given *if one* (or two, or all of them) library is missing.
Does your text say "*if all libraries* libraries", doesn't it?

File: scripts/qa-vdb [hunk 3]
```diff
@@ -67,6 +67,10 @@ usage <<-EOU
 	Missing slots won't be displayed when a binding operator (:=) is used as
 	portage records the slot even if it was missing from the ebuild.
 
+	No warning is given in the event libraries are missing entirely or not
```
Comment (ionenwks) [inline scripts/qa-vdb:70]: > But... No warning is given _if one_ (or two, or all of them) library is missing. Does your text say "_if all libraries_ libraries", doesn't it?

Implied "any libraries" rater than all, but could reword it a bit to be clearer (and get rid of the event thing like at it), maybe:
```suggestion
	No warning is given for libraries that are missing entirely or not
```
Applies to any libraries that weren't found regardless of amount.

Comment (kuraga) [reply scripts/qa-vdb:70]: Aha, I see: `entirely or not`. Strange but I saw `entirely` only :smile:

Comment (kuraga) [reply scripts/qa-vdb]: Aha, I see: `entirely or not`. Strange but I saw `entirely` only :smile: 
`entirely or not, associated` vs `entirely, or not associated`, hm...

Ok, my speech was about `entirely` word.

**Review by ionenwks — APPROVED** [review]

**Comment by ionenwks** [comment]
Merged in commit 2bbc8409b8913a9f846108d5665d0243d442e837, thanks.

**Pull request closed without merging**